CONTRIBUTING.adoc

```diff
@@ -96,8 +96,9 @@ NOTE: Replace "linux/amd64" with the appropriate value for your platform/archite
 
 10.  Launch another terminal, change into the same directory you started OpenShift, and deploy the private docker registry within OpenShift with the following commands:
 
-        $ sudo chmod +r openshift.local.config/master/admin.kubeconfig
-        $ ./oadm registry -n default --config=openshift.local.config/master/admin.kubeconfig
+        $ sudo chmod +r openshift.local.config/master/admin.kubeconfig openshift.local.config/master/openshift-registry.kubeconfig
+        $ ./oadm registry -n default --config=openshift.local.config/master/admin.kubeconfig \
+        	--credentials=openshift.local.config/master/openshift-registry.kubeconfig
```

Comment:
ah thanks and what about the serviceaccount ? It does not seem to be created (or documented)